luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/LuthierIndustrialPark.java

```diff
@@ -213,24 +271,17 @@ public ResourceDictionaries getDictionaries() {
          * @param resourceDictionaries  a class that contains resource dictionaries including
          * PhysicalTableDictionary, DimensionDictionary, etc.
          */
-        public Builder(LuthierResourceDictionaries resourceDictionaries) {
+        public Builder(
+                LuthierResourceDictionaries resourceDictionaries
+        ) {
             this.resourceDictionaries = resourceDictionaries;
             dimensionFactories = getDefaultDimensionFactories();
             keyValueStoreFactories = getDefaultKeyValueStoreFactories();
             searchProviderFactories = getDefaultSearchProviderFactories();
             physicalTableFactories = getDefaultPhysicalTableFactories();
+            logicalTableGroupFactories = getDefaultLogicalTableGroupFactories();
         }
 
-        /**
```

Comment:
Whey are we removing this constructor?

Reply:
This builder is not used anywhere in the internal unit testing. And it is not more superior than the one above it so we don't use it in the LuthierBinderFactory. 
It should not be used by the user since the contract is that user only initialize a LIP through (some subclass of) luthier binder factory. 
There is no point to keep it in my opinion, unless we think a bean-style initialization is going to be interesting. 
I'd like to hear about it in that case @archolewa

Reply:
This is fine then.